core/mondoo-edr-policy.mql.yaml

```diff
@@ -184,3 +185,9 @@ queries:
     mql: |
       service('EraAgentSvc').running
       service('EraAgentSvc').enabled
+  - uid: mondoo-edr-policy-ensure-defender-agent-is-running-windows
+    filters: |
+      asset.family.contains('windows')
```

Comment:
In this case the asset filter is missing the check for service('WinDefend').installed otherwise it will be checked with all other variants

Reply:
All right, added, I assume you mean all other variants from other policies. Thanks